.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,80 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+
+# Workflow name
+name: "chromatic"
+
+on:
+  issue_comment:
+    types: [created]
+
+# List of jobs
+jobs:
+  visual-testing:
+    # Operating System
+    runs-on: ubuntu-latest
+
+    # Job steps
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+
+      - uses: 8BitJonny/gh-get-current-pr@1.1.0
+        id: getPr
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Only return if PR is still open
+          filterOutClosed: true
+
+      - run: echo "The current PR number is ${prNumber}"
+        if: success() && steps.getPr.outputs.number
+        env:
+          prNumber: ${{ steps.getPr.outputs.number }}
+
+      - name: Get most recent comment
+        if: success() && steps.getPr.outputs.number
+        uses: peter-evans/find-comment@v1
+        id: latestComment
+        with:
+          issue-number: ${{ steps.getPr.outputs.number }}
+          body-includes: ""
+          direction: last
+
+      - run: |
+          echo "runChromatic=${runChromatic}" >> $GITHUB_ENV
+        env:
+          runChromatic: ${{ contains(steps.latestComment.outputs.comment-body, 'run chromatic') }}
```

Comment:
/nit Something like `@chromatic run` is a little more extensible and would allow us to write new command should we need them in the future. I like the way Dependabot has approached it: 
```
@dependabot rebase will rebase this PR
@dependabot recreate will recreate this PR, overwriting any edits that have been made to it
@dependabot merge will merge this PR after your CI passes on it
... etc
```

Reply:
I wasn't sure about that, because `@dependabot` actually at-mentions the dependabot account/user. I think @chromatic would at-mention chromatic, and they'd get a bunch of notifications. We could do `@cultureamp`, but I wasn't sure if that would also generate a bunch of notifications we don't want.

Reply:
> I think @chromatic would at-mention chromatic (this person: https://github.com/chromatic), and they'd get a bunch of notifications.

Confirmed! No harm done though.

Reply:
Yeah, good call (and thanks for chiming in @chromatic 😄). We also have a @cultureamp-ci account that we can use https://github.com/cultureamp-ci. @jasononeil or @zioroboco might have some thoughts on whether a PR bot for common actions is something we might want to invest time in and make available for other repos.

Reply:
(That being said, I don't think this conversation should block this PR and would be happy for this to be merged with `chromatic run` as the trigger)